CHANGELOG.md

```diff
@@ -13,12 +13,14 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
+- Fixing base images to multi-architecture lists. ([#534])
```

Comment:
```suggestion
- Fixing base images to multi-architecture lists ([#534]).
- airflow and superset: Pull statsd-exporter as binary from Nexus instead of extracting out of the official docker image ([#534]).
```